wercker.yml

```diff
@@ -28,6 +30,27 @@ ci:
         code: |
           bash <(curl -s https://codecov.io/bash)
 
+# build and test UI
+ci-ui:
+  box: node
+  steps:
+    - script:
+        name: install and build
+        code: |
+          cd ui
+          npm install
+          npm build
```

Comment:
Ah, if we want to run the build command we need to run `npm run build`. `npm build` is an internal command that npm uses during install and other stuff